src/java.base/share/classes/java/nio/Bits.java

```diff
@@ -151,40 +167,53 @@ static void reserveMemory(long size, long cap) {
             // DirectBufferAllocTest to (usually) succeed, while
             // without it that test likely fails.  Since failure here
             // ends in OOME, there's no need to hurry.
-            long sleepTime = 1;
-            int sleeps = 0;
-            while (true) {
-                if (tryReserveMemory(size, cap)) {
-                    return;
-                }
-                if (sleeps >= MAX_SLEEPS) {
-                    break;
-                }
+            for (int sleeps = 0; true; ) {
```

Comment:
More typical coding pattern in openjdk code.  Here and elsewhere in this PR.
```suggestion
            while (true) {
                int sleeps = 0;
```

Reply:
That's not the same thing, and doesn't do what's needed here.  Perhaps you meant
```
int sleeps = 0;
while (true) {
```
I like limiting the scope of the variable.  Is that a suggestion or a request to change?